pages/faq/troubleshooting/faq.md

```diff
@@ -138,6 +139,10 @@ Discontinued devices will no longer be actively supported by {{ $names.company.l
 
 There are a few options for devices that do not have an official device type on {{ $names.company.lower }}. If your device has an x86 architecture, you can try the [Intel NUC][nuc] image, which is built to support generic x86 devices. For other devices, you can [build your own][build-your-own] version of {{ $names.os.lower }} using our [open source repos][balenaOS]. To discuss custom board support, please contact sales@{{ $names.email_domain }}.
 
+##### What to keep in mind while buying or using power supply units or PSU's?
```

Comment:
```suggestion
##### What to keep in mind while buying or using power supply units (PSUs)?
```

Reply:
Didn't go for bracket short form, because on top in the TOC it might have acted weirdly. Removed the short form altogether from the question.